### ISSUE TITLE
Typo(repoName) : accessiblity-testing-backend

### ISSUE THREAD

**Issue by DivyanshuVortex** [ruxailab/accessiblity-testing-backend#31]
Rename repository to fix typo

Current name:
`accessiblity-testing-backend`

Correct name:
`accessibility-testing-backend`

The repository name currently contains a spelling mistake ("accessiblity").  
It should be updated to **accessibility-testing-backend** to reflect the correct spelling and maintain consistency across repositories .
